reactor-core/src/main/java/reactor/core/publisher/MonoCompletionStage.java

```diff
@@ -62,8 +68,6 @@ public Object scanUnsafe(Attr key) {
     }
 
     static class MonoCompletionStageSubscription<T> implements InnerProducer<T>,
-                                                               Fuseable,
-                                                               QueueSubscription<T>,
```

Comment:
Note, `MonoCompletionStage` also has `Fuseable` removed. They are not needed, as the fusion support is `NONE`. And it makes the subscriber wrapping to the restoring subscriber possible (it is not `Fuseable`).